docs/pages/tutorials/quick-start-java.md

```diff
@@ -8,7 +8,7 @@ This quick start guide introduces key concepts that we will use for modeling rea
 In this tutorial, we'll build our `helloworld` service using the Java interface. The `helloworld` service will be composed of a single instance of the `helloworld` pod, running a single `server` task.
 
 ### Step 0 - Clone the project
-Get started by forking: https://github.com/mesosphere/dcos-commons and cloning it on your workstation. Change your working directory to `dcos-commons`.
+Get started by forking: [https://github.com/mesosphere/dcos-commons](https://github.com/mesosphere/dcos-commons) and cloning it on your workstation. Change your working directory to `dcos-commons`.
 
 ```bash
 $ cd dcos-commons
```

Comment:
I deliberately didn't add it because we're asking the user to fork the repo so that they can push back to it (so the git remote is going to be different to the default dcos-commons one). In the long run, this should be brought in line with the developer-guide.md document.